linux_os/guide/services/ssh/ssh_server/sshd_set_idle_timeout/rule.yml

```diff
@@ -67,6 +68,9 @@ ocil: |-
     <pre>ClientAliveInterval {{{ xccdf_value("sshd_idle_timeout_value") }}}</pre>
 
 warnings:
-    - dependency: |-
-        SSH disconnecting idle clients will not have desired effect without also
-        configuring ClientAliveCountMax in the SSH service configuration.
+    - general: |-
+        Following conditions may prevent the SSH session to time out:
+        <ul>
+        <li>Remote processes on the remote machine generates output. As the output has to be transferred over the network to the client, the timeout is reset every time such transfer happens.</li>
+        <li>Any <tt>scp</tt> or <tt>sftp</tt> activity by the same user to the host resets the timeout.</li>
+        </ul>
```

Comment:
`requires:` needs to be added to the rule for ClientAliveCountMax

Reply:
Would you propose to introduce a new rule, or to alter an existing one? We have a rule for `ClientAliveCountMax`, but that one is parametrized, and it may set the value of that configuration option to other values than 0.

Reply:
Then the rule for ClientAliveCountMax needs to be updated instead of a new rule.